artifactory/commands/transferfiles/state/utils.go

```diff
@@ -16,6 +16,11 @@ const (
 	secondsInMinute = 60
 	secondsInHour   = 60 * secondsInMinute
 	secondsInDay    = 24 * secondsInHour
+
+	OldTransferDirectoryStructureErrorMsg = "unsupported transfer directory structure found.\n" +
+		"This structure was created by a previous run of the transfer-files command, but is no longer supported by this JFrog CLI version.\n" +
+		"You may either downgrade JFrog CLI to the version that was used before, or remove the transfer directory which is located under the JFrog CLI home directory\n" +
```

Comment:
"located under the JFrog CLI home directory"
Can we elaborate more about the exact path? I'd prefer that users will be able to work around issues by themselves